wifinina/wifinina_nano-rp2040.go

```diff
@@ -0,0 +1,13 @@
+//go:build nano_rp2040
+// +build nano_rp2040
+
+package wifinina
+
+// Defines pins RGB LED is connected to on Arduino Nano RP2040 Connect board
+// See https://docs.arduino.cc/static/3525d638b5c76a2d19588d6b41cd02a0/ABX00053-full-pinout.pdf
+
+const (
+	LED_R = 27
```

Comment:
I know this is not subject to future change, but should these definitions not be taken from `machine` package, i.e: `LED_R = machine.GP27` ?

Reply:
Tricky here. You see, these PIN numbers are not of RP2040 itself, but NINA-W102 chip.
Perhaps this shall be extracted to main tinygo repo, to `board_nano-rp2040.go` with names like `NINA_LED_R`.
But that shall be thought through properly, to avoid confusion and people trying to use these leds as built-in led, in fact accessing RP2040 pins instead of NINA's
See here https://docs.arduino.cc/static/3525d638b5c76a2d19588d6b41cd02a0/ABX00053-full-pinout.pdf